README.md

```diff
@@ -5,6 +5,7 @@
 - CMake 3.12+
 - A C Compiler
 - pkg-config
+- time command
```

Comment:
maybe explicitly say `/usr/bin/time`, not built-in time

Reply:
Updated, and I also changed the message for checking for /usr/bin/time in the CMakeLists.txt in the tests directory so the CMake output looks nicer now. The latest commit addressing this should be squashed into the the previous commit before this is merged.